### ISSUE TITLE
Find Raw View item with FindElementByAccessibilityId method

### ISSUE THREAD

**Issue by bpawluk** [microsoft/WinAppDriver#757]
I am unable to access UI elements which have AutomationProperties.AccessibilityView property set as "Raw" using FindElementByAccessibilityId method. Is it possible to search elements from Raw Tree view using WinAppDriver?

More on the view trees concept: https://docs.microsoft.com/windows/uwp/design/accessibility/basic-accessibility-information#influencing-the-ui-automation-tree-views
